index.html

```diff
@@ -327,14 +327,11 @@ <h2>About me</h2>
             required
           ></textarea>
 
-
+          <button type="button" class="link">Get in touch</button>
```

Comment:
- Please use the correct type on the button to submit the form. The click action is not working.